dev_guide/deployments.adoc

```diff
@@ -275,8 +274,8 @@ specified in the deployment configuration's `helloworld` container, a new
 deployment is created using the new image for the `helloworld` container.
 
 NOTE: If an ImageChange trigger is defined on a deployment configuration (with
-a ConfigChange trigger or with automatic=true) and the ImageStreamTag pointed
-by the ImageChange trigger does not exist yet then the first deployment will
+a ConfigChange trigger or with `automatic=true`) and the ImageStreamTag pointed
+by the ImageChange trigger does not exist yet, then the first deployment will
```

Comment:
@tnguyen-rh I know this was there already, but I'd get rid of the "will":

"...then the first deployment automatically starts as soon as..."

But also, does ImageStreamTag and ImageChange need markup?